test/assertions/assertConvertToBuffer.js

```diff
@@ -24,10 +22,8 @@ const assertConvertToBuffer = async (data) => {
         }
     }
     catch (err) {
-        sut = err;
+        expect(err).to.not.exist();
```

Comment:
Moving the expectation in side of the catch will have the side effect of `expect(..)` not running for all cases.

Reply:
yup which is fine since the error can only exist to begin with in the catch, otherwise it's an expect that we know is null